fn_ansible_tower/entrypoint.sh

```diff
@@ -7,12 +7,10 @@ APP_CONFIG_SHA=`sha256sum /etc/rescircuits/app.config`
 resilient-circuits run & CIRCUITS_PID=`echo $!`
 while true
 do
-  echo -n "`date +'%F %T,%N INFO '`"
   if ! echo $APP_CONFIG_SHA | sha256sum --check
   then
     kill -9 $CIRCUITS_PID
     break
   fi
-  if [ "$CIRCUITS_PID" != "`ps -o pid= -p $CIRCUITS_PID|xargs`" ]; then break; fi
```

Comment:
these need to remain

Reply:
okay, but have not modified this file, may be there file diff in appexchange and github